less/common/Search.less

```diff
@@ -61,11 +62,21 @@
     .transition(all 0.4s);
     box-sizing: inherit !important;
   }
+
+  .LoadingIndicator-container {
+    height: 36px;
+  }
+
   .Button {
     float: left;
     margin-left: -36px;
-    width: 36px !important;
     outline: none;
+    width: 36px !important;
+
+    &.LoadingIndicator {
+      width: var(--size) !important;
+      padding: 0;
+    }
```

Comment:
I don't think this is used since LoadingIndicator is a child here.

Reply:
It's not. We make the loading indicator take the Button class and a few more classes for... questionable reasons. This replaces the normal "clear field" button while sources are loading.